corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/controller.js

```diff
@@ -119,6 +121,28 @@ FormplayerFrontend.module("Menus", function (Menus, FormplayerFrontend, Backbone
         },
 
         getDetailList: function (detailObject) {
+
+            if (detailObject.get('entities')) {
+                var entities = detailObject.get('entities');
+                var listModel = [];
+                // we need to map the details and headers JSON to a list for a Backbone Collection
+                for (var i = 0; i < entities.length; i++) {
+                    var listObj = {};
```

Comment:
nit: these variables get hoisted: https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Statements/var#var_hoisting. which is why vars are usually declared at the top